java/bulk-export/bulk-export-planner/src/main/java/sleeper/bulkexport/planner/SqsBulkExportProcessorLambda.java

```diff
@@ -58,20 +59,39 @@ public class SqsBulkExportProcessorLambda implements RequestHandler<SQSEvent, Vo
 
     public SqsBulkExportProcessorLambda() throws ObjectFactoryException {
         this(AmazonSQSClientBuilder.defaultClient(), AmazonS3ClientBuilder.defaultClient(),
-                AmazonDynamoDBClientBuilder.defaultClient());
+                AmazonDynamoDBClientBuilder.defaultClient(), System.getenv(CONFIG_BUCKET.toEnvironmentVariable()));
     }
 
-    public SqsBulkExportProcessorLambda(AmazonSQS sqsClient, AmazonS3 s3Client, AmazonDynamoDB dynamoClient)
+    public SqsBulkExportProcessorLambda(
+            SqsBulkExportProcessor processor,
+            BulkExportQuerySerDe bulkExportQuerySerDe,
+            InstanceProperties instanceProperties,
+            AmazonSQS sqsClient,
+            AmazonS3 s3Client,
+            AmazonDynamoDB dynamoClient) {
+        this.processor = processor;
+        this.bulkExportQuerySerDe = bulkExportQuerySerDe;
+        this.instanceProperties = instanceProperties;
+        this.sqsClient = sqsClient;
+        this.s3Client = s3Client;
+        this.dynamoClient = dynamoClient;
+    }
+
+    public SqsBulkExportProcessorLambda(AmazonSQS sqsClient, AmazonS3 s3Client, AmazonDynamoDB dynamoClient, String configBucket)
+            throws ObjectFactoryException {
+        this(sqsClient, s3Client, dynamoClient, configBucket, S3TableProperties.createProvider(
+                S3InstanceProperties.loadFromBucket(s3Client, configBucket), s3Client, dynamoClient));
+        instanceProperties = S3InstanceProperties.loadFromBucket(s3Client, configBucket);
+    }
+
+    public SqsBulkExportProcessorLambda(AmazonSQS sqsClient, AmazonS3 s3Client, AmazonDynamoDB dynamoClient, String configBucket, TablePropertiesProvider tablePropertiesProvider)
```

Comment:
It doesn't look like this constructor is used, except in the constructor immediately above. It might be better to remove it and just have the one above?